docs/create-packages/Symbol-Packages-V2.md

```diff
@@ -0,0 +1,96 @@
+---
+title: How to publish NuGet symbol packages using the new symbol package format '.snupkg'| Microsoft Docs
+author:
+- cristinamanu
+- kraigb
+ms.author:
+- cristinamanu
+- kraigb
+manager: skofman
+ms.date: 10/30/2018
+ms.topic: reference
+ms.prod: nuget
+ms.technology: null
+description: How to create NuGet packages that contain only symbols to support debugging of other NuGet packages in Visual Studio.
```

Comment:
"How to create NuGet Symbols packages to support debugging of the package in Visual Studio"

The way this reads now is a little confusing.

Reply:
Updated to: How to create NuGet symbol packages (snupkg).